### PR TITLE
Feature/14 input field and create user

### PR DESCRIPTION
**Pull request by shivansh-pandeyy**

### CODE REVIEW

File: src/views/Users/CreateUser/CreateUser.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+      <Stack spacing={2} component='form' onSubmit={handleSubmit} >
+        <Input onChange={handleChange}
+          onBlur={handleBlur}
+          id={fieldNames.NAME}
+          label="Name"
+          errorText={errors.name}
+          error={errors.name ? true : false}
+          value={values.name} 
+          />
```
Comment (anand-velotio) [inline src/views/Users/CreateUser/CreateUser.tsx]: This is being re-used multiple times in this file. Can you loop over the list of fields and use this component only once ?

File: src/views/Users/CreateUser/CreateUser.tsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+          label="Website"
+          errorText={errors.website} error={errors.website ? true : false} value={values.website} 
+          />
+        <Input
+          onChange={handleChange}
+          onBlur={handleBlur}
+          id={fieldNames.EMAIL}
+          label="Email"
+          errorText={errors.email}
+          error={errors.email ? true : false}
```
Comment (anand-velotio) [inline src/views/Users/CreateUser/CreateUser.tsx]:
```suggestion
          error={!!errors.email}
```

Comment (anand-velotio) [reply src/views/Users/CreateUser/CreateUser.tsx]: At all the other places as well

File: src/views/Users/CreateUser/index.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import CreateUser from './CreateUser';
+
+export default CreateUser;
```
Comment (anand-velotio) [inline src/views/Users/CreateUser/index.ts]: Either use the index.tsx or CreateUser.tsx. No need to maintain both everywhere in the project

File: src/views/Users/Users.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { Box, Stack, Typography } from '@mui/material'
+import React from 'react'
+import UserCard from '../../components/UserCard'
+import Placeholder from '../../assets/images/user-placeholder.png';
+import Add from '../../assets/images/add1.png';
+import { Link } from 'react-router-dom';
+
+const Users: React.FC<{}> = () => {
```
Comment (anand-velotio) [inline src/views/Users/Users.tsx]: add return type of Users component

**Review by anand-velotio — CHANGES_REQUESTED** [review]

File: src/views/Users/CreateUser/CreateUser.tsx (hunk 1 above)
Comment (shivansh-pandeyy) [reply src/views/Users/CreateUser/CreateUser.tsx]: Done

File: src/views/Users/CreateUser/CreateUser.tsx (hunk 2 above)
Comment (shivansh-pandeyy) [reply src/views/Users/CreateUser/CreateUser.tsx]: done

File: src/views/Users/CreateUser/index.ts (hunk 3 above)
Comment (shivansh-pandeyy) [reply src/views/Users/CreateUser/index.ts]: done

File: src/views/Users/Users.tsx (hunk 4 above)
Comment (shivansh-pandeyy) [reply src/views/Users/Users.tsx]: done

**Pull request merged**